ex9/phonebook.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style/style.css">
+  <script src="js/lib/jquery-3.1.1.js"></script>
+  <script src="js/lib/jquery.tmpl.js"></script>
+  <script data-main="js/main" src="js/lib/require.js"></script>
+</head>
+<body>
+  <input type="button" id="btnReloadList" value="Reload" />
+  <input type="button" id="btnASC" value="ASC" />
+  <input type="button" id="btnDESC" value="DESC" hidden />
+
+  <input type="button" id="btnSearchOn" value="Search" />
+  <input type="text" id="textSearch" value="" hidden />
+  <input type="button" id="btnFilter" value="Filter" hidden />
+  <input type="button" id="btnSearchOff" value="Exit" hidden/>
+
+  <script id="phoneBookTemplate" type="text/template">
+    <!-- It loops an array -->
+    <li>
+      <span class="name">${name}</span>
+      <span class="phone">
+        <a href="tel:${comm.phone}">${comm.phone}</a>
+      </span>
+      <span class="email">
+        <a href="emailto:${comm.mail}">${comm.mail}</a>
+      </span>
+    </li>
+  </script>
+
+  <ol id="phoneBook">
+    <li>
+      <span class="name fname">
```

Comment:
Now that you actually load the data dynamically, I'd remove the pre-filled values.